game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/ForumPosterEditorViewModel.java

```diff
@@ -88,10 +88,10 @@ synchronized void setForumSelection(final String forumSelection) {
         this.forumSelection.equals(NodeBbForumPoster.TRIPLEA_FORUM_DISPLAY_NAME)
             ? ClientSetting.tripleaForumUsername.getValue().map(String::valueOf).orElse("")
             : ClientSetting.aaForumUsername.getValue().map(String::valueOf).orElse("");
-    forumPasswordIsSet =
+    forumPasswordIsSet = false; /*
```

Comment:
If I understand correctly, we'll use 'username' + 'password' to create a token via API, then store that token but not the password (correct?).

If so, I think we probably want this field to check if the token is set. If the token is set, then we can fill in the password field with a dummy value to make it look like there is something filled in (that should be essentially what we have now - the reason for that is so we don't have to read the password from settings and set the actual value to the text field. Instead we use a dummy string to show that the field is set).